From b8a8179bbed4fe6b4ac5d0e8d886ac6ece985b8f Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Thu, 31 Mar 2022 13:19:16 +0200
Subject: [PATCH] Remove test-only dependencies

---
 setup.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/setup.py b/setup.py
index 95a533e..042127e 100644
--- a/setup.py
+++ b/setup.py
@@ -156,11 +156,7 @@ setup(
                       "networkx",
                       "numpy",
                       "vtk",
-                      "cython",
-                      "GitPython",
                       "configparser",
-                      "testtools",
-                      "pypandoc",
                       "sympy>=1.7",
                       "scipy"],
     ext_modules=exts,
-- 
2.35.1

